mysql-test/suite/innodb/t/tablespace_encrypt_9.test

```diff
@@ -79,7 +79,7 @@ SET GLOBAL innodb_master_thread_disabled_debug = 1;
 --error 0,CR_SERVER_LOST,ER_INTERNAL_ERROR
```

Comment:
What is the path of this file in 5.7 HEAD?

Reply:
This is a new 8.0 file but the problem is the same sync_latch_meta_init() memory leak.